### PR TITLE
PostgreSQL - tag popular module error

### PR DESCRIPTION
**Pull request by alikon**
#### Steps to reproduce the issue

Set the tag popular module parameter Order  to random

Click on Popular Tags item from the fronted left menu All Modules
#### Actual result

you got a sql error message
popular tags
#### How to tests

Apply the pacth
#### Addictional Comments

similar to #8238,#8240
was used `RAND()`  instead of  the api `$query->Rand()`

### CODE REVIEW

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on 103abeddf4c7ac53d65541cfbb46940854004eeb<br /><br />Works good here on mysql. Thanks.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/8242">issues.joomla.org/joomla-cms/8242</a>.</sub>

**Comment by wilsonge** [comment]
You can't change the parameter value like this. It will break people who have already saved the existing parameter

File: modules/mod_tags_popular/helper.php [hunk 1]
```diff
@@ -81,8 +71,18 @@ public static function getList(&$params)
 		}
 
 		$query->join('INNER', $db->quoteName('#__tags', 't') . ' ON ' . $db->quoteName('tag_id') . ' = t.id')
-			->join('INNER', $db->quoteName('#__ucm_content', 'c') . ' ON ' . $db->quoteName('m.core_content_id') . ' = ' . $db->quoteName('c.core_content_id'))
-			->order($order_value . ' ' . $order_direction);
+		->join('INNER', $db->qn('#__ucm_content', 'c') . ' ON ' . $db->qn('m.core_content_id') . ' = ' . $db->qn('c.core_content_id'));
+
+		if ($order_value == 'random')
```
Comment (zero-24) [inline modules/mod_tags_popular/helper.php]: `if ($order_value == 'random' or $order_value == 'rand()')` for B/C as @wilsonge posted

**Comment by alikon** [comment]
param unchanged now

**Comment by waader** [comment]
I have tested this item :white_check_mark: successfully on 0c875a5ef3b4338ff2c30eb8b637287697be3864<br /><br />Thanks alikon!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/8242">issues.joomla.org/joomla-cms/8242</a>.</sub>

**Pull request merged**